docs/concepts/multi-chain-proposals.md

```diff
@@ -0,0 +1,693 @@
+---
+id: multi-chain-proposals
+title: Multi-Chain Proposals
```

Comment:
[arbitrary comment location]

currently "multichain proposals" is listed on the nav after "Glossary." But it should be in "Governance," right?

<img width="273" alt="Screenshot 2025-01-10 at 10 30 08 AM" src="https://github.com/user-attachments/assets/315c632e-dbb1-40db-87e4-f172f7f447e8" />

Reply:
Done: c92c9da1c18c344ee66c52113ef08d3d591d08d2